chat-plugins/panagrams-beta.js

```diff
@@ -152,4 +152,132 @@ exports.commands = {
+				if (guessName in Game.guessed) return user.sendTo(this, 'That word has already been guessed.');
+				if (guessName === (Game.answer.id || toId(Game.answer.species))) {
+					this.add('|html|<b>' + user.name + '</b> guessed <b>' + (guess.name || guess.species) + '</b>, which was the correct answer! Congratulations!');
+					Game.end();
+				} else if (guess.num < 1 || guess.baseSpecies) user.sendTo(this, (guess.name || guess.species) + ' is either an alternate form or doesn\'t exist in the games. These cannot be guessed.');
+				else {
+					if (Object.keys(Game.users).length < 3) {
+						this.add('|html|<b>' + user.name + '</b> guessed <b>' + (guess.name || guess.species) + '</b>, but was not the correct answer...');
+						Game.guessed[guessName] = user.userid;
+					} else {
+						this.add('|html|<b>' + user.name + '</b> guessed <b>' + (guess.name || guess.species) + '</b>, but was not the correct answer. You have failed to guess the Pokemon, which was <b>' + (answer.name || answer.species) + '</b>');
+						Game.end();
+					}
+				}
+			} else {
+				if (message && message !== true) {
+					this.add('|c|' + user.getIdentity(this.id) + '|' + message);
+				}
+			}
+			this.update();
+		};
+	}
+	Panagram.prototype.end = function (forced) {
+		if (forced) this.room.add('The game of panagram has been forcibly ended.');
+		this.room.chat = Rooms.Room.prototype.chat;
+		delete pGames[this.room.id];
+	};
+	return Panagram;
+})();
+
+var cmds = {
+	rules: 'help',
+	help: function (target, room, user) {
+		if (!this.canBroadcast()) return;
+		this.sendReplyBox('<b>Panagram Commands</b><br>' +
+			'<li>/panagram <em>Category</em> - Starts a game of panagram that follows the specified category. Valid categories include <i>Pokemon, Items, Abilities</i> and <i>moves</i>. Not specifying a category starts a pokemon panagram by default. Requires + or higher to use.' +
+			'<li>/panagram end <small>or</small> /endp - Ends a game of panagram. Requires + or higher to use.<br>' +
+			'Users can guess answers by simply typing them into the chat.'
+		);
+	},
+
+	pokemon: '',
+	item: '',
+	items: '',
+	ability: '',
+	abilities: '',
+	move: '',
+	moves: '',
+	'': function (target, room, user, connection, cmd) {
+		if (pGames[room.id]) return this.sendReply('There is currently a game of panagram going on in this room.');
+		if (!this.can('broadcast', null, room)) return this.sendReply('You must be ranked + or higher to start a game of panagram in this room.');
+		if (!cmd) cmd = 'pokemon';
+
+		pGames[room.id] = new Panagram(room, cmd);
+	},
+
+	end: function (target, room, user) {
+		if (!pGames[room.id]) return this.sendReply('There is no game of panagram going on in this room.');
+		if (!this.can('broadcast', null, room)) return this.sendReply('You must be ranked + or higher to end a game of panagram in this room.');
```

Comment:
We can do 1 of 2 things here:
1. Use ``this.errorReply();``
2. Just have it ``return false;``